src/component/panels/IntegralsPanel/IntegralPanel.tsx

```diff
@@ -195,6 +193,34 @@ function IntegralPanelInner({
           <IntegralsPreferences ref={settingRef} />
         )}
       </div>
+      <ConfirmModal
+        headerColor="red"
+        onConfirm={confirmHandler}
+        onCancel={() => {
+          close();
+        }}
+        isOpen={isOpen}
+        onRequestClose={() => {
+          close();
+        }}
+        saveText="Yes"
+        cancelText="No"
+      >
+        <div
+          style={{
+            display: 'flex',
+            flex: '1 1 0%',
+            alignItems: 'center',
+            justifyContent: 'center',
+            fontWeight: 'bold',
+            margin: 25,
+            fontSize: 14,
+            color: 'rgb(175, 0, 0)',
+          }}
+        >
```

Comment:
This is not ok. You're repeating the same styles over and over.
Either we accept the new style, or we improve the style of the component in react-science.

Reply:
Okay I will try to remove it

Reply:
With styling: 
![image](https://github.com/cheminfo/nmrium/assets/56769876/5cdcdcfd-955c-49c8-b89b-7d50d3139be9)

Without:
![image](https://github.com/cheminfo/nmrium/assets/56769876/f0b73c8e-a1db-42dd-857f-d8068b7e7850)

I think we should update `ConfirmModal` style in r`eact-science`

Reply:
the needed style
```tsx
style={
            fontWeight: 'bold',
            margin: 25,
            fontSize: 14,
            color: 'rgb(175, 0, 0)',
}
```

Reply:
No, make a PR to `react-science` (with stories if needed)